<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>拖放功能组件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Top">
     <item>
      <widget class="QGroupBox" name="groupBox_setObject">
       <property name="title">
        <string>设置对象</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="radioButton_listSource">
          <property name="text">
           <string>listSource</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_listWidget">
          <property name="text">
           <string>listWidget</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_treeWidget">
          <property name="text">
           <string>treeWidget</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_tableWidget">
          <property name="text">
           <string>tableWidget</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_params">
       <property name="title">
        <string>拖放参数</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBox_acceptDrops">
          <property name="text">
           <string>acceptDrops</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_dragEnabled">
          <property name="text">
           <string>dragEnabled</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_dragDropMode">
          <property name="text">
           <string>dragDropMode</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="comboBox_DropAction">
          <item>
           <property name="text">
            <string>NoDragDrop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DragOnly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DropOnly</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DragDrop</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>intemalMove</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_dragDropMode_2">
          <property name="text">
           <string>defaultDropAction</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="comboBox_CopyAction">
          <item>
           <property name="text">
            <string>CopyAction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MoveAction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LinkAction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>IgnoreAction</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>258</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Bottom">
     <item>
      <widget class="QGroupBox" name="groupBox_listSource">
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>listSource</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget_listSource">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>剪切</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>复制</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>粘贴</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>粗体</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>斜体</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>下划线</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>左对齐</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>中间对齐</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>右对齐</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>红色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>绿色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蓝色</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_listWidget">
       <property name="title">
        <string>listWidget</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_treeWidget">
       <property name="title">
        <string>treeWidget</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_tableWidget">
       <property name="title">
        <string>tableWidget</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
